### PR TITLE
Prepare release 0.2.0

### PR DESCRIPTION
**Pull request by lionel-**

### CODE REVIEW

**Comment by hadley** [comment]
Do you want to alphabetise the NEWS first?

**Comment by lionel-** [comment]
What do you mean?

**Comment by hadley** [comment]
https://style.tidyverse.org/news.html#news-release

**Comment by lionel-** [comment]
> Within a section, bullets should be ordered alphabetically by the first function mentioned. If no function is mentioned, place at the top of the section.

:white_check_mark:

File: NEWS.md [hunk 1]
```diff
@@ -131,86 +129,87 @@ or possible cast.
...
 * New `.names_to` parameter for `vec_rbind()`. If supplied, this
   should be the name of a column where the names of the inputs are
   copied. This is similar to the `.id` parameter of
   `dplyr::bind_rows()`.
 
-* `vec_c()`, `vec_rbind()`, and `vec_cbind()` gain a `.name_repair`
-  argument (#227, #229).
+* New `vec_seq_along()` and `vec_init_along()` create useful sequences (#189).
+
+* `vec_slice()` now supports row names.
```
Comment (hadley) [inline NEWS.md]:
```suggestion
* `vec_slice()` now preserves character row names, if present.
```

File: NEWS.md [hunk 2]
```diff
@@ -131,86 +129,87 @@ or possible cast.
...
-* New `vec_seq_along()` and `vec_init_along()` create useful sequences (#189).
+* `vec_slice()` can correct slice 0-row data frames (#179).
 
-* `vec_equal_na()` now returns `TRUE` for data frames and records when
-  every component is mising, not when _any_ component is missing
-  (#201).
+* New `vec_repeat()` for repeating each element of a vector the same number
+  of times.
 
 * `vec_type2(x, data.frame())` ensures that the returned object has
```
Comment (hadley) [inline NEWS.md]: I might've just missed it, but do you describe the improvements to error messages somewhere?

File: _pkgdown.yml [hunk 3]
```diff
@@ -6,19 +6,44 @@ reference:
...
+  - vec_recycle
+
+- title: Equality and ordering
+  contents:
+  - vec_equal
+  - vec_compare
+
+- title: Sorting and splitting
+  contents:
+  - vec_split
```
Comment (hadley) [inline _pkgdown.yml]: I think split should do with unique

File: NEWS.md (hunk 2 above)
Comment (lionel-) [reply NEWS.md]: You're thinking about error messages when nested data frames are not congruent right?

**Pull request merged**